### PR TITLE
LUN-161: Test for confirmation modal

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by klinzo — APPROVED** [review]

File: src/components/__tests__/ConfirmationModal.spec.tsx [hunk 1]
```diff
@@ -1,45 +1,43 @@
...
-      const modalProps: ConfirmationModalPropsType = {
-        ...defaultModalProps,
-        visible: true
-      }
+    it('should close on cancel button click', () => {
+      const { getByText } = render(<ConfirmationModal {...defaultModalProps} />, { wrapper: wrapWithTheme })
+      const cancelButton = getByText('cancel')
+      fireEvent.press(cancelButton)
+      expect(setVisible).toBeCalledWith(false)
+      expect(confirmationAction).not.toBeCalled()
```
Comment (klinzo) [inline src/components/__tests__/ConfirmationModal.spec.tsx:33]: I think you could assert that `navigation.navigate` was called as well with the right params.

Comment (SteffiStuffel) [reply src/components/__tests__/ConfirmationModal.spec.tsx:33]: Navigation is not called anymore. The `ConfirmationModal` is now more generic and can be used for different purposes, e.g. also the confirmation when deleting a module, where no navigation is needed. Therefore the confirmation-action is now passed as a prop and can (but needn't) trigger a navigation

**Comment by klinzo** [comment]
Still one review needed :)

**Comment by SteffiStuffel** [comment]
@Taggotty

**Review by Taggotty — APPROVED** [review]

**Pull request merged**